src/charm.py

```diff
@@ -406,6 +532,12 @@ def _on_pfcp_agent_pebble_ready(self, event: EventBase) -> None:
         """Handle pfcp agent Pebble ready event."""
         if not self.unit.is_leader():
             return
+        if invalid_configs := self._get_invalid_configs():
+            self.unit.status = BlockedStatus(
+                f"The following configurations are not valid: {invalid_configs}"
+            )
+            event.defer()
```

Comment:
I do not think the defer here is necessary, as `config_changed` will be emitted before this is fixed.

Reply:
Not exactly sure what you mean here. If we don't defer, then how do we make sure pfcp actually starts after the configs are fixed? We run this code only on `pfcp_agent_pebble_ready`. 
Should we run pfcp configuration on `config_changed` as well (as we already do for bess)?

Reply:
Looking a bit more at the code, I think we can just drop the checks for the invalid config in this hook, as the pfcp container does not use any configurations.

Reply:
Indeed. Removed.